.github/workflows/test.yml

```diff
@@ -21,8 +21,8 @@ jobs:
               "dev4/x86_64-el9-clang16-opt",
               "dev4/x86_64-el9-gcc13-opt",
               "LCG_107/x86_64-el9-clang16-opt",
-              "LCG_107/x86_64-el9-gcc13-opt",  # root 6.34.02 !minimal root version for RNTuple
-              "LCG_104/x86_64-el9-gcc13-opt"]  # root 6.28.04 !minimal root version check
+              "LCG_107/x86_64-el9-gcc13-opt"]  # root 6.34.02 !minimal root version for RNTuple
```

Comment:
Can we keep an environment with ROOT 6.28.04 since that is our minimum version we might require (if many features are turned off). IIUC this is mainly changed here to make CI pass? Can we solve this in a way where we don't drop this version check?